### DISCUSSION TITLE
Die ersten Computerspiele

### DISCUSSION THREAD

**Discussion by eltikia** [eltikia/eltikia.github.io#1] [Q&A] [answered]
Mit welschen Computerspiele fingst du an?

**Comment by Stagyrite** [comment, marked as answer]
Im Jahr 2023 ich hatte vier C64-Spiele bereit.

1. [Zak McKracken and the Alien Mindbenders](https://www.youtube.com/watch?v=S3ETogTxtnY "🇩🇪 Zak McKracken and the Alien Mindbenders - YouTube")
2. [Maniac Mansion](https://www.youtube.com/watch?v=UlD1jRwcvhA "🇩🇪 Maniac Mansion - YouTube")
3. [The Mask of the Sun](https://www.youtube.com/watch?v=6ZJ_S4K6R10 "🇩🇪 The Mask of the Sun - YouTube")
4. [Das Drachental](https://www.youtube.com/watch?v=pahNjOWqNxQ "🇩🇪 Das Drachental - YouTube")

Das war genug zu dieser [Internetseite](https://eltikia.github.io/ "Nostalgie-Ecke von Stagirit") gebaut. Ich musste nicht sehr schnell sein. Ich musste nicht sehr vorsichtig sein. Aber der Zweck war erreicht.
